github-connector/internal/kymasender/kyma_sender_test.go

```diff
@@ -0,0 +1,78 @@
+package kymasender
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/apperrors"
+
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/eventparser"
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/eventparser/mocks"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type toJSON struct {
+	TestJSON string `json:TestJSON"`
+}
+
+type ClientMock struct {
+}
+
+func (c *ClientMock) Do(req *http.Request) (*http.Response, error) {
+	return &http.Response{}, nil
+}
+
+func TestSendToKyma_ProperArguments(t *testing.T) {
+	t.Run("should return no error", func(t *testing.T) {
+		k := NewWrapper(&ClientMock{}, eventparser.NewEventParser())
```

Comment:
You don't validate whether event service was called. The test should check it the call is performed correctly. You could use the mock, or take a slightly different approach. Instead of passing interface you could use http.client struct and use httptest.NewServer to create http server for testing purposes.